src/mam4xx/mo_setinv.hpp

```diff
@@ -0,0 +1,123 @@
+#ifndef MAM4XX_MO_SETINV_HPP
+#define MAM4XX_MO_SETINV_HPP
+
+#include <haero/atmosphere.hpp>
+
+#include <mam4xx/aero_config.hpp>
+#include <mam4xx/mam4_types.hpp>
+
+namespace mam4 {
+namespace mo_setinv {
+// number of invariants
+static const int nfs = 8;
+static const int num_tracer_cnst = 4;
+
+struct Config {
+  // FIXME: BAD CONSTANTS!
+  // conversion factor for Pascals to dyne/cm^2
+  Real Pa_xfac = 10.0;
+  // presumably, the boltzman constant, in CGS units
+  // NOTE: passed in via yaml input file for now
+  Real boltz_cgs = 0.13806500000000001E-015;
```

Comment:
I'm not wild about making this a configurable parameter, but that hardly distinguishes this particular piece of code!

Reply:
Turns out the `NOTE` was an extraneous leftover--not passed in via yaml.

The local cgs Boltzman constant, on the other hand... 😶